app/assets/stylesheets/application.scss

```diff
@@ -1,14 +1,14 @@
-/*
- *= require jquery.ui.autocomplete
- *= require vendor
- *= require scoped_search
- *= require multi-select
- *= require select2
- *= require select2-bootstrap
- *= require_self
- *= require_tree .
- */
-
+@import "jquery.ui.core";
+@import "jquery.ui.menu";
+@import "jquery.ui.theme";
+@import "jquery.ui.autocomplete";
```

Comment:
@ohadlevy @dLobatog 
the 3 imports above are required by autocomplete, when importing //=require function won't work, so i just imported all of the needed files myself.